pkg/utils/gitrepo.go

```diff
@@ -827,7 +827,7 @@ func sortKubeResource(crdsAndNamespaceFiles, rbacFiles, otherFiles []string, pat
 			if err != nil {
 				klog.Warning("Failed to unmarshal YAML file")
 				// Just ignore the YAML
-				return crdsAndNamespaceFiles, rbacFiles, otherFiles, nil
+				return crdsAndNamespaceFiles, rbacFiles, otherFiles, err
```

Comment:
This might not be correct since there is a warning up front and and it might be intentional that we are not returning the err here.

If you want to use the err then put it in the warning message is suffice. 
If you think it should return with err, maybe a separate PR is better.